config/config.exs

```diff
@@ -24,24 +24,14 @@ import Config
 config :realtime_signs,
   http_client: HTTPoison,
   posts_log_dir: "log/posts/",
-  sign_head_end_host: "127.0.0.1",
-  sign_ui_url: "signs-dev.mbtace.com",
-  sign_ui_api_key: nil,
   time_zone: "America/New_York",
-  trip_update_url: "https://s3.amazonaws.com/mbta-gtfs-s3/rtr/TripUpdates_enhanced.json",
-  vehicle_positions_url:
-    "https://s3.amazonaws.com/mbta-gtfs-s3/rtr/VehiclePositions_enhanced.json",
   sign_updater_mod: PaEss.Logger,
   http_poster_mod: HTTPoison,
   scheduled_headway_requester: Headway.Request,
   headway_calculator: Headway.HeadwayDisplay,
   external_config_getter: ExternalConfig.Local,
   aws_client: ExAws,
   s3_client: ExAws.S3,
-  s3_bucket: nil,
-  s3_path: nil,
-  api_v3_key: nil,
-  api_v3_url: "https://api-dev-green.mbtace.com",
   number_of_http_updaters: 4,
```

Comment:
Since this is now written in the runtime config, we should remove it from here.

Reply:
Ah good catch! So this one we actually do need to keep for this PR because of another bug. Basically, in `http_updater.ex`, this config is actually read at compile time rather than runtime, even though it is treated in `realtime_signs.ex` like a runtime config value so it still needs a value for `number_of_http_updaters` at compile time.

I have a PR almost ready for making that change at #554 , but I still need to remove this config value, so I'll do it over in that PR!